content/docs/studio/how-it-works.md

```diff
@@ -1,48 +1,54 @@
 # How Iterative Studio Works
 
-[Iterative Studio](https://studio.iterative.ai/) works with the data, metrics,
-hyperparameters and model metadata that you add to your ML project Git
-repositories. It works very closely with your Git ecosystem.
+[Iterative Studio](https://studio.iterative.ai/) works very closely with your
+Git ecosystem. It parses the data, metrics, hyperparameters and model metadata
+that you add to your ML project Git repositories, and renders them on a web UI.
```

Comment:
This change is unrelated in this PR, but since it was updated, here's a suggestion because the current wording is unclear, esp. around the concept of metadata:

```suggestion
[Iterative Studio](https://studio.iterative.ai/) works very closely with your
Git ecosystem. It parses the metadata found in your ML project repositories to
present organized data, hyperparameters, models, and metrics on a web UI.
```